src/get_maintainer_wrapper.sh

```diff
@@ -38,40 +38,62 @@ function print_files_authors()
 
 # Executes get_maintainer with the given file or dir
 #
-# @1 First argument can be "--authors" or "a" to enable authors
-#    printing. It is optional.
-# @2 Second argument is the given file/dir. It is optional and, if not
-#    given, the current working directory will be used.
+# @raw_options String with all user options
 #
 # Returns:
 # False uppon error and true otherwise.
 #
 # This function also handle the cases where:
-# - A patch file is given
 # - The working dir and/or given path is not the root of a linux kernel repo
 # - The working dir and/or given path is not inside a linux kernel repo
+# - Option --authors/-a is given. It will print the name of the module's
+# authors
+# - Option --update-patch/-u is given. It will include a "To:" field in
+# the header of the patch with the contact of the maintainers associated
+# with that patch
```

Comment:
Nit-pick:
I recommend you to ident the second line with the first character of the bullet.